plugins/module_utils/clickhouse.py

```diff
@@ -37,6 +37,7 @@ def client_common_argument_spec():
         login_user=dict(type='str', default=None),
         login_password=dict(type='str', default=None, no_log=True),
         client_kwargs=dict(type='dict', default={}),
+        flatten_nested=dict(type='int', default=1),
```

Comment:
```suggestion
        flatten_nested=dict(type='int'),
```
please move this to plugins/modules/clickhouse_client.py too